src/ResponseTagger.php

```diff
@@ -36,10 +43,24 @@ class ResponseTagger
      * Constructor
      *
      * @param TagsInterface $client
+     * @param array         $options
```

Comment:
the text part of the phpdoc should list the supported options and explain them (terse phrase is ok, something like "strict: if true, throws exceptions on empty tags")